src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -482,6 +488,235 @@ describe('526 helpers', () => {
     });
   });
 });
+describe('transform', () => {
+  const transformedMinimalData = {
```

Comment:
@SMLuthi is going to take a look at these to make sure they pass validation.

Reply:
Like validation in the schema?

Reply:
Yeah, on the vets-api side.